src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+}
+
+.product-card__product-image {
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__product-name {
+  margin-bottom: 4px;
+  width: 166px;
+  height: 36px;
+  line-height: 18px;
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.product-card__product-code {
+  align-self: flex-start;
+  margin-bottom: 16px;
+  width: 98px;
+  height: 14px;
+  line-height: 14px;
+  font-size: 10px;
+  color: #616070;
+}
+
+.product-rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-self: stretch;
+}
+
+.product-card__product-rating {
+  margin-bottom: 24px;
+}
+
+.product-rating__stars {
```

Comment:
Stars is a separate block. Please move all the styles for start to separate file stars.css
Also it would be great if you created a file for card block too. And in style.css you leave only global styles

Reply:
Done